### DISCUSSION TITLE
How do I fix not connecting to HTTPS ?

### DISCUSSION THREAD

**Discussion by IkelAtomig** [coddingtonbear/obsidian-local-rest-api#47] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/101938856/209638771-4c11b0b2-d690-4806-b545-dae286398a80.png)
I don't want to use insecure HTTP server port.

**Comment by IkelAtomig** [comment]
Any insights on this ?

**Comment by coddingtonbear** [comment, marked as answer]
This unfortunately varies by operating system.  To get to an answer, I recommend searching on google for how to add a "Trusted Certificate Authority" (AKA "CA")  in your environment.

**Reply by IkelAtomig** [reply]
Okay, What certifcate should I use ?

Ain't it possible to do without adding a CA manually.

**Reply by coddingtonbear** [reply]
You can download a certificate from the settings page for the plugin.  No -- trusting the certificate is not possible without adding a CA manually.

**Reply by IkelAtomig** [reply]
I copied the Root certifcate and saved it as `.crt` and installed it using this page - https://deliciousbrains.com/ssl-certificate-authority-for-local-https-development/. 

On inputting the API key, I am connected securely via API. I would suggest you to add information about this.

**Reply by IkelAtomig** [reply]
The Only problem is MS edge can't trust the CA even though Brave does. Help regarding this would be great.

**Reply by coddingtonbear** [reply]
Definitely feel free to write something on the wiki about that!

**Reply by tocram1** [reply]
Just a heads up a few years later... Your `renderServerUrlTable` function which contains a link to a wiki article about adding a trusted root certificate... which doesn't work anymore :/ (it points to this [404 URL](https://github.com/coddingtonbear/obsidian-web/wiki/How-do-I-get-my-browser-trust-my-Obsidian-Local-REST-API-certificate%3F)).
Maybe you didn't port said wiki article from another old repo to this repo..?
